src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -707,6 +707,26 @@ private static FlowAnalysisAnnotations DecodeFlowAnalysisAttributes(PEModule mod
             return annotations;
         }
 
+        internal override ImmutableHashSet<string> NotNullIfParameterNotNull
+        {
+            get
+            {
+                // TODO2 cache?
```

Comment:
Please address or file a bug.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/37233](https://github.com/dotnet/roslyn/issues/37233)

---
In reply to: [303572191](https://github.com/dotnet/roslyn/pull/37201#discussion_r303572191) [](ancestors = 303572191)